### ISSUE TITLE
Segfault in vcl_backend_response

### ISSUE THREAD

**Issue by gauthier-delacroix** [varnish/libvmod-var#5]
Hi, 

I get segfault as soon as I use `libvmod-var` in `vcl_backend_response` with Varnish 4.0.0.

Default VCL with just changing this:
````
sub vcl_backend_response {
  var.set("foo", "bar");
}
````

Gives this:
````
kernel: varnishd[31186]: segfault at 18 ip 00007fa2431fc28f sp 00007fa2420ce170 error 4 in libvmod_var.so[7fa2431fb000+3000]
kernel: [81589.491201] varnishd[31186]: segfault at 18 ip 00007fa2431fc28f sp 00007fa2420ce170 error 4 in libvmod_var.so[7fa2431fb000+3000]
````

Doing the same in `vcl_recv` works fine.

**Comment by carlosabalde** [comment]
Hi,

I've checking latest changes in Varnish 4.x sources and -if I understood correctly-, `PRIV_REQ`/`PRIV_SESS` have been discarded and replaced by `PRIV_TASK` (https://www.varnish-cache.org/patchwork/patch/233/), which basically is a local thread/request storage, but separate for fronted and backend threads.  Does that mean this issue could not be fixed in order to have access to variables in `vcl_backend_*`?

I have a similar issue in another VMOD and I'm curious about the plans here :)

**Comment by aondio** [comment]
This has been fixed in 4.1
There is a workaround  for 4.0, the README points to this issue for explanations and quick fix.
